src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -347,7 +347,7 @@ private void HandleActorTaskSchedulerMessage(ActorTaskSchedulerMessage m)
         /// </summary>
         /// <param name="mailbox">TBD</param>
         /// <returns>TBD</returns>
-        internal Mailbox SwapMailbox(Mailbox mailbox)
+        internal Mailbox? SwapMailbox(Mailbox mailbox)
```

Comment:
The `Mailbox` could be, in theory, `null` if the compare-and-swap operation fails - should never happen.